synnax/pkg/access/action/action.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 Synnax Labs, Inc.
+//
+// Use of this software is governed by the Business Source License included in the file
+// licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with the Business Source
+// License, use of this software will be governed by the Apache License, Version 2.0,
+// included in the file licenses/APL.txt.
+
+// package action provides a list of common actions that can be performed in API
+// requests to the server. Resource-specific actions are defined in the respective
+// packages for each resource.
+package action
```

Comment:
Why does this require a separate package?

Reply:
There was a circular dependency from some separate actions in the `ontology` package in the distribution layer.

Reply:
Ok, gotcha